### PR TITLE
Store the subs color in UMS.conf as the hexa string

### PR DESCRIPTION
**Pull request by valib**
+ fix the logging when the renderer's maximum bitrate setting has the
priority over the user's setting

This fix issues #1149 and #1150

### CODE REVIEW

File: src/main/java/net/pms/encoders/MEncoderVideo.java [hunk 1]
```diff
@@ -1408,8 +1409,8 @@ public ProcessWrapper launchTranscode(
 
 				if (override_ass_style) {
 					String assSubColor = "ffffff00";
-					if (configuration.getSubsColor() != 0) {
-						assSubColor = Integer.toHexString(configuration.getSubsColor());
+					if (!configuration.getSubsColor().isEmpty()) {
```
Comment (Sami32) [inline src/main/java/net/pms/encoders/MEncoderVideo.java]: @valib Why this way is better ?
Could not an NPE be throwed ?

**Comment by Sami32** [comment]
What a busy bee :+1: 
Nice work :-)

**Comment by Sami32** [comment]
@valib I thought that since it's RGB 0xffffff was fine.
> Reading subtitles_color: "ffff33cc" (default: "ffffffff")
Did the extra byte used for alpha channel ?

**Comment by Sami32** [comment]
@valib No idea ?
0xFFFFFF       (RGB)
0xFFFFFFFF     ?

**Comment by valib** [comment]
@Sami32 this is the Java presentation of the sRGB color see https://docs.oracle.com/javase/7/docs/api/java/awt/Color.html which includes also alpha channel and this color format is used also with FFmpeg.

**Comment by Sami32** [comment]
@valib Thank you very much :smile: 
I was only suspecting it, as my stupid brain was stuck to HTML code representation...

Honestly, i don't think that users will ever change it from their configuration file.
But if they want keep a safe copy of their general configuration for later fast ready use just in case, why not...

**Pull request merged**